packages/application-shell/src/components/fetch-project/fetch-project.spec.tsx

```diff
@@ -43,7 +43,7 @@ describe('rendering', () => {
     it('should fetch project and pass data to children function', async () => {
       renderProject();
 
-      await waitForElementToBeRemoved(() => screen.getByText('loading...'));
+      await waitForElementToBeRemoved(() => screen.queryByText('loading...'));
```

Comment:
```suggestion
      await waitForElementToBeRemoved(screen.queryByText('loading...'));
```

Reply:
Interesting rule change. In my head I always felt it would be easier for a negated `waitFor*` implementation to wait for a DOM selector function not to throw.